src/features/join/hooks/useUserHook.ts

```
@@ -32,26 +33,36 @@ export const useAgreeTerms = () => {
 // 인증번호 발급
 export const useSendCertificationCode = () => {
   return useMutation({
-    mutationFn: (phoneNum: string) => sendCertificationCode(phoneNum),
+    mutationFn: (data: { phoneNumber: string }) => sendCertificationCode(data),
     onSuccess: () => {
       alert('인증번호를 발송했습니다.');
     },
-    onError: () => {
-      alert('인증번호 전송에 실패했습니다.');
+    onError: (error: AxiosError<any>) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)